### PR TITLE
Small improvement in the installation script

### PR DESCRIPTION
**Pull request by PetarTodorovv**
**Description**
Invoke `patchJWKS` after compass DB installation to minimise the unnecessary logs. Also, increased sleep interval in the same function.

**Pull Request status**

- [x] Implementation
- [ ] Unit tests
- [ ] Integration tests
- [ ] `chart/compass/values.yaml` is updated 
- [ ] Mocks are regenerated, using the automated script

### CODE REVIEW

File: installation/scripts/install-db.sh [hunk 1]
```diff
@@ -51,7 +51,7 @@ do
 done
 set -- "${POSITIONAL[@]}" # restore positional parameters
 
-echo "Install DB"
+echo "Installing DB"
 helm upgrade --install --atomic --debug --timeout "${TIMEOUT}" -f ./mergedOverrides.yaml --create-namespace --namespace compass-system localdb "${DB_CHARTS}"
```
Comment (ognyvrac) [inline installation/scripts/install-db.sh:55]: As you are already removing the spam from the logs, maybe remove the `--debug` flag to exclude verbose output

Comment (PetarTodorovv) [reply installation/scripts/install-db.sh]: not sure what was the initial motivation to use this flag but when I compared the logs most of the debug logs are about the helm charts/values.yaml. there are couple with resource creation info. Nevertheless, I remove the debug flag.

**Review by ognyvrac — APPROVED** [review]

**Review by la4ezar — APPROVED** [review]

**Pull request merged**